lib/services/access_list.go

```diff
@@ -297,3 +297,68 @@ func SelectNextReviewDate(accessList *accesslist.AccessList) time.Time {
 
 	return nextDate
 }
+
+// AccessListReviewsGetter defines an interface for reading access list reviews.
+type AccessListReviewsGetter interface {
+	// ListAccessListReviews will list access list reviews for a particular access list.
+	ListAccessListReviews(ctx context.Context, accessList string, pageSize int, pageToken string) (reviews []*accesslist.Review, nextToken string, err error)
+}
+
+// AccessListReviews defines an interface for managing Access List reviews.
+type AccessListReviews interface {
+	AccessListReviewsGetter
```

Comment:
It's more advantageous to have this separated than not IMO -- the separate interface helps with the cache object.

Reply:
While I don't disagree, this pattern seems to be pretty well established. It could also add value to the watchers, if we need an access list review watcher in the future.

Reply:
Fair enough. I've removed this.